packages/backend/src/index.ts

```diff
@@ -86,7 +86,11 @@ async function main() {
   const config = await loadBackendConfig({
     argv: process.argv,
     logger,
+    remote: {
+      reloadIntervalSeconds: 60 * 60 * 12, // Check remote config changes every 12 hours. Change to your desired interval in seconds
```

Comment:
Let's keep it disabled in the example backend tbh, don't really see the need to promote it here and it takes up valuable space :grin: